test/tansl.spec.coffee

```diff
@@ -0,0 +1,154 @@
+chai = require 'chai'
+should = chai.should()
+assert = chai.assert
+path = require 'path'
+
+Translation = require '../src/ts/i18n.ts'
+# process.env.DEBUG = 'bt-player-ui:*'
+
+swallowConsoleWarnings = false
+
+try
+	describe 'Translation tests: ', ->
```

Comment:
Please write the tests in TypeScript as well... I can see how the CS syntax is much cleaner but all our projects are currently pure TypeScript and introducing another language would slow us down and increase the complexity

Reply:
Btw. great that you wrote tests 👍